conf/test.routes

```diff
@@ -3,3 +3,5 @@ POST    /etmp/reporting-window-state                                  uk.gov.hmr
 GET     /etmp/reporting-window-state                                  uk.gov.hmrc.disareturnsstubs.testonly.controllers.EtmpTestOnlyController.getReportingWindowState
 
 POST    /etmp/open-obligation-status/:isaManagerReferenceNumber       uk.gov.hmrc.disareturnsstubs.testonly.controllers.EtmpTestOnlyController.openObligationStatus(isaManagerReferenceNumber: String)
+
+POST    /:isaManagerReferenceNumber/:year/:month/reconciliation       uk.gov.hmrc.disareturnsstubs.testonly.controllers.GenerateReportController.create(isaManagerReferenceNumber: String, year: String, month: String)
```

Comment:
do we want an `nps` prefix to match shape of other endpoints?

Reply:
Hmmm for this one, I'm not sure it is specifically stubbing an NPS endpoint. My understanding(whether that is correct or not who knows) is that this endpoint is to allow test reports to be generated. Then we will have the NPS GET report endpoint that disa-returns will call into to fetch that generated report

Reply:
good point!